08_Intro_jQuery/solution_student/select_fun/js/main.js

```diff
@@ -30,11 +38,13 @@
 */
 
 // 6. Hide the item with the id #dropdownMenu
-
+$('#dropdownMenu').hide();
 
 // 7. When the element with the id #dropdownButton is clicked,
 	// a) slide toggle #dropdownMenu
-
+$('##dropdownButton').on('click', function() {
```

Comment:
So close! You wrote it all right and had the correct logic, you just accidentally put two '#'s in your selector! That's why it didn't work: `$('##dropdownButton')`